Engine/Module/ModuleTime.cpp

```diff
@@ -27,6 +27,10 @@ bool ModuleTime::Init()
 
 	real_time_clock->Start();
 
+#if GAME
```

Comment:
The game clocks is not needed in the editor?

Reply:
It is, but it starts wehen play is clicked